content/en/serverless/step_functions/troubleshooting.md

```diff
@@ -35,7 +35,7 @@ If your organization has an existing all-encompassing index with a low limit, pl
 - Verify that you can see both Lambda traces and Step Function traces in Datadog.
 - Verify that you are using Python layer v75+ or Node.js layer v94+.
```

Comment:
@kimi-p  Should me also update the Python and Node.js layer version for the 128 bits change?

Reply:
This is done in the previous PR https://github.com/DataDog/documentation/pull/23546